File: typescript/src/ejercicios/ejercicio06.ts
(function () {
    let base = 0;
    let altura = 0;
    base = parseInt(prompt("Base: "));
    altura = parseInt(prompt("Altura: "));
    let area = base * altura;
    console.log("el area es:" + area);
    Arearectangulo(base, altura);

    function Arearectangulo(base, altura) {
        return base * altura;
    }

})();

File: java/co.edu.sena.capitulo90/src/main/java/co/edu/sena/capitulo9/Ejercicio06.java
package co.edu.sena.capitulo9;

import java.util.Scanner;

public class Ejercicio06 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		double base = 0;
		double altura = 0;
		System.out.print("Base: ");
		base = teclado.nextFloat();
		System.out.print("Altura: ");
		altura = teclado.nextFloat();
		double area = base * altura;
		System.out.println("el area es:" + area);
		rectangulo(base, altura);
	}

	public static double rectangulo(double base, double altura) {
		return base * altura;
	}
}